frontend/src/routes/index.tsx

```diff
@@ -115,6 +115,16 @@ export default component$(() => {
       handleShortenerClick$();
     }
   });
+  
+const changeBtnVisibility$ = $((e: Event) => { 
+  const inputValue = e?.target?.value;
+  const shortnerBtn = document?.getElementById("shortenerBtn");
```

Comment:
Hey, I would prefer to use a state to store the input.

Please check out: https://qwik.builder.io/docs/components/state/

Reply:
@origranot Made suggested change please review again